src/sidebar/components/MentionSuggestionsPopover.tsx

```diff
@@ -99,7 +99,7 @@ export default function MentionSuggestionsPopover({
   ...popoverProps
 }: MentionSuggestionsPopoverProps) {
   return (
-    <Popover {...popoverProps} classes="p-1">
+    <Popover {...popoverProps} classes="p-1 !max-h-96">
```

Comment:
The PR explains the rationale clearly, but I would add a brief comment in the code as well.